File: src/java/com/distributedlife/mahjong/reference/permute/PungPermutator.java
package com.distributedlife.mahjong.reference.permute;

import java.util.List;

class PungPermutator extends MultiplePermutator {
    public PungPermutator(List<String> tilesPungIsAllowedIn, String suit) {
        super(tilesPungIsAllowedIn, 3, suit);
    }
}

File: lib/permute/pung-permutator.js
/**
 * PungPermutator - Handles permutations for pungs (3 tiles)
 */
import { MultiplePermutator } from './multiple-permutator.js';

export class PungPermutator extends MultiplePermutator {
  constructor(tilesPungIsAllowedIn, suit) {
    super(tilesPungIsAllowedIn, 3, suit);
  }
}
